notebooks/recommender-system.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
<p>I just have some clarifying questions: _might help the tutorial clarity too_ </p><p><code>X = params[: num_movies * num_features].reshape</code></p><p> </p><p>Why does X need a reshape? Can you define it as num_movies x num_features?</p><p> </p><p><code>theta = params[num_movies * num_features:].reshape</code></p><p>Similar, isn't this taking the original 1680x943 and shaping it into 1680x943 again?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>